nix/iohk-nix-src.json

```diff
@@ -0,0 +1,7 @@
+{
+  "url": "https://github.com/input-output-hk/iohk-nix",
+  "rev": "0a34a05d611a7141d2b6d73446be004fa06f0f5c",
```

Comment:
Update `haskell.nix` to the latest revision on `master` and nix-shell development environments should work. @jbgi